### ISSUE TITLE
Problems with the search bar with the compact mode on zen browser

### ISSUE THREAD

**Issue by santigids** [zen-browser/desktop#3544]
### Captchas

- [ ] I have read the instructions.
- [ ] I have searched existing issues and avoided creating duplicates.
- [ ] I am not filing an enhancement request.

### What happened?

![Captura de pantalla 2024-12-11 010927](https://github.com/user-attachments/assets/d83d8e03-4e8b-4845-94e6-511f0d82be6f)
![Captura de pantalla 2024-12-11 011156](https://github.com/user-attachments/assets/d275a842-86e6-419b-92a6-f61fdfb0901d)
when i use the compact mode on zen browser, the search bar is bugged when you write on this, but when this mode is disable y all ok.

### Reproducible?

- [x] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.0 (64-bit)

### What platform are you seeing the problem on?

Windows

**Comment by BlueCyro** [comment]
Hate to bump this with what is essentially a "me too", but I can confirm at the very least that this issue isn't isolated. It also happens on Linux (EndeavourOS, Arch-based) when I have compact mode enabled. It appears to even be a little bit more exacerbated on my side since I can't even click the search bar settings or star button either.

![image](https://github.com/user-attachments/assets/0fa24aab-0bcc-4a99-ad59-0228c4dffdae)

**Comment by MrPiggy105** [comment]
I don't see any bugged behaviour that doesn't happen outside of compact mode. In my testing, the issue where you can't click anything on the bar without it unfocusing happens outside of compact mode too. Other than that, idk what you mean by bugged

**Comment by PRIME12602** [comment]
Yeah, i have the same bug and whats with the spacing on the vertical tabs? i mean the home button used to be visible but now it covers everything. 
take a look at the left top corner, it wasnt like this. 
![{A2DC9B66-1A56-48F0-A8FD-83F897F03221}](https://github.com/user-attachments/assets/00c08d93-d970-4576-bb0f-b1e8042dfd32)

**Comment by onlyspxctre** [comment]
Is there an option to disable this new search bar design in the settings or `about:config`? Facing the same issue here (tested on Windows 11 & Gentoo Linux).

**Comment by JSGRD22** [comment]
Has this been resolved

**Comment by vkpdeveloper** [comment]
I am facing the same issue.

**Comment by vkpdeveloper** [comment]
In the latest version of Zen I am not facing any issues with the Search bar, it's smooth and I am using the compact mode already.
